dangerfile.ts

```diff
@@ -32,15 +32,30 @@ if (packageChanged && !lockfileChanged) {
   warn(`${message} - <i>${idea}</i>`);
 }
 
-// ensure we have access to github for this check
+// Ensure we have access to github for these checks
 let isTrivial = false;
+let isYarnAuditMissing = false;
 if (danger.github) {
-  isTrivial = includes((danger.github.pr.body + danger.github.pr.title), "#trivial")
+  let prBody = danger.github.pr.body
+  
+  isTrivial = includes((prBody + danger.github.pr.title), '#trivial')
+    
+  if (lockfileChanged) {
```

Comment:
I wonder if this will be the case for all dependabot PRs.. we should probably skip all checks if the user is dependabot, if you want to add that here or we can do it in another PR